maps/site53/items/bullets.dm

```diff
@@ -111,36 +111,36 @@
 	desc = "A 5.7x28mm bullet casing."
 	caliber = "10mm"
 	projectile_type = /obj/item/projectile/bullet/pistol/medium/smg
-	icon_state = "10mm-casing"
-	spent_icon = "10mm-casing-spent"
+	icon_state = "pistol-casing"
+	spent_icon = "pistol-casing-spent"
 
 /obj/item/ammo_casing/rub10mm
 	desc = "A rubber 10mm bullet casing."
 	caliber = "10mm"
 	projectile_type = /obj/item/projectile/bullet/pistol/medium/smg/rubber
-	icon_state = "10mm-casing-rubber"
-	spent_icon = "10mm-casing-spent"
+	icon_state = "pistol-casing"
+	spent_icon = "pistol-casing-spent"
```

Comment:
Please replace it with somewhat equivalent icon states if they exist, for instance `pistol-casing-rubber` here, go through the icons and see if any more variants exist

Reply:
Aye. Changed this one because the 10mm casing icon is just big and kinda ugly

Reply:
Excellent idea and by coincidence, the exact one you mention actually exist

Reply:
> Excellent idea and by coincidence, the exact one you mention actually exist
No coincidence, I just looked up and saw smg bullets already used the icon